apps/backend/src/db/migrations/20240918040515_add_task_notes_table.ts

```diff
@@ -0,0 +1,37 @@
+import { type Knex } from "knex";
```

Comment:
We already have a migration to add this table in main, it was added in a backend part of your task